sonic-syseepromd/scripts/syseepromd

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python2
+
+'''
+    syseepromd
+    Syseeprom information gathering daemon for SONiC
+    This daemon will be started during the start phase of pmon container, gathering syseeprom info and write to state DB.
+    It will continue monitoring the state DB for the syseeprom table, if table been deleted, it will write again.
+    With this daemon, show syseeprom CLI will be able to get data from state DB instead of access hw or cache.
+'''
+
+try:
+    import signal
+    import threading
+    from sonic_daemon_base.daemon_base import DaemonBase
+    from sonic_daemon_base.daemon_base import Logger
+    from sonic_daemon_base import daemon_base
+    from swsscommon import swsscommon
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+PLATFORM_SPECIFIC_MODULE_NAME = 'eeprom'
+PLATFORM_SPECIFIC_CLASS_NAME = 'board'
+
+EEPROM_INFO_UPDATE_PERIOD_SECS = 10
+
+POST_EEPROM_SUCCESS = 0
+ERR_PLATFORM_NOT_SUPPORT = 1
+ERR_FAILED_EEPROM = 2
+ERR_FAILED_UPDATE_DB = 3
+ERR_INVALID_PARAMETER = 4
+ERR_EEPROMUTIL_LOAD = 5 
+
+EEPROM_TABLE_NAME = 'EEPROM_INFO'
+SYSLOG_IDENTIFIER = 'syseepromd'
+
+# Global logger class instance
+logger = Logger(SYSLOG_IDENTIFIER)
+
+class DaemonSyseeprom(DaemonBase):
+    def __init__(self):
+        DaemonBase.__init__(self)
+
+        self.stop = threading.Event()
```

Comment:
I think the name "stop" can make the code a bit confusing. Suggest rename to "stop_event"

Reply:
renamed.